test/fixtures/tags.yml

```diff
@@ -78,4 +78,8 @@ co-author:
 
 unpublished-note-tag:
   tid: 20
-  name: unpublished-note-tag
\ No newline at end of file
+  name: unpublished-note-tag
+  
+ awesome_cool:
```

Comment:
Ah but actually we want to have the test return if one node has two separate tags, "awesome" and "cool" -- given a single query "awesome cool" -- make sense?

Reply:
I think so.. see my new changes :)